app/views/browse/service.html

```diff
@@ -62,50 +62,48 @@
                       <dd>{{service.spec.type}}</dd>
                       <dt>IP:</dt>
                       <dd>{{service.spec.clusterIP}}</dd>
+                      <dt ng-if-start="service.spec.externalName">External Name</dt>
+                      <dd ng-if-end>{{service.spec.externalName}}</dd>
```

Comment:
@jwforres Added this field to display a service's external name (if any) for the new types of services. https://tcms-openshift.rhcloud.com/case/6047/

PTAL